### ISSUE TITLE
Predict Problem

### ISSUE THREAD

**Issue by suencgo** [MIC-DKFZ/nnUNet#1513]
Thanks for your @FabianIsensee  excellent work-the nnunet!
When I have finished the training and run the predict, I have faced a problem.
I am trying to run my own network structure in nnunetv1.  Since my task is a two-category task, the dimension of my output is (1, 2, 512, 512) but he asked to accept (1, 1, 512, 512) when predicting, so I use this.
(slice_output,_ = torch.max(outputs[-1],dim=1,keepdim=True))
The training process is very smooth. During the predicting process, the output of my network is, 

![3c58267861b99e56ea8fef7796aa56d](https://github.com/MIC-DKFZ/nnUNet/assets/105877597/7502cffc-adfb-4840-84fe-b409920c613a)

but the ret obtained through my trainer file (inherited from TrainerV2) is indeed, where seg and The value of softmax is, 

![01dd463bd176a9cf5f35d2f3bf392a4](https://github.com/MIC-DKFZ/nnUNet/assets/105877597/792abeb8-3225-4e96-8b06-10152d7e616b)

this proves that nnunet did not accept the correct output tensor, what is the problem? Looking forward your reply!!!!!!

**Comment by Kobalt93** [comment]
Hi suencgo,
It is really hard to say something without debugging your modifications, but we are happy to raise some potential error possibilities that might help you resolve the issue you're facing:
- Did you do this modification only during training? I suggest making your modification also inference.
- torch.max -> torch.argmax?
